dev/Projections/Directory.Build.props

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE-CODE in the project root for license information. -->
+<!-- This is a special MSBuild file that is parsed before everything else when MSBuild detects it in our directory structure.
+     Adding Sdk="Microsoft.NET.Sdk" to a project causes NuGet files to be generated very early on, so we need these defines to be here
+     to ensure that all of our build output files end up in the same location. -->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\Directory.Build.props" />
+
+  <PropertyGroup>
+    <!-- Both the IID optimizer and the CsWinRT lookup table aren't needed with the new CsWinRT version,
+         so disabling them here until they are disabled automatically by CsWinRT. -->
+    <CsWinRTIIDOptimizerOptOut>true</CsWinRTIIDOptimizerOptOut>
+    <CsWinRTCcwLookupTableGeneratorEnabled>false</CsWinRTCcwLookupTableGeneratorEnabled>
```

Comment:
Yes, once I get the next CsWinRT update (after current preview one), this would not be needed anymore.   I mainly put this just here because this repo produces multiple projections and the same thing is generated in each one when it is not needed.